webui/react/config-overrides.js

```diff
@@ -52,5 +54,26 @@ module.exports = override(
       'process.env.VERSION': '"0.12.12.dev0"',
       'process.env.IS_DEV': JSON.stringify(process.env.NODE_ENV === 'development'),
     })
-  )
+  ),
+
+  addWebpackPlugin(
+    new MonacoWebpackPlugin({
+      // available options are documented at https://github.com/Microsoft/monaco-editor-webpack-plugin#options
+      languages: ['yaml'],
+      features: [
+        'suggest',
+        'quickOutline',
+        'wordHighlighter',
+        'colorDetector',
+        'codelens',
+        'parameterHints'
+      ],
+    })
+  ),
 );
+
+module.exports = {
+  webpack: webpackConfig,
+  jest: (config, env) => ({...config, ...jestConfig}),
+  devServer: undefined,
```

Comment:
**question:** what does setting `devServer` to `undefined` do? Is this needed for something specific?

Reply:
nope just added it there to let us see it's available. could be a comment